Service/UserApi/DocumentService.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\ClientHelperBundle\Service\UserApi;
+
+final class DocumentService extends UserApiService
+{
+    /**
+     * @param array<string, mixed> $context
+     * @return array<mixed>
+     */
```

Comment:
```suggestion
```

The $context array will dissapear with other remarks (will be the request directly).
Why return an array? You will need a response later on, so why not return it in your service.